pkg/await/apps_deployment.go

```diff
@@ -580,6 +663,10 @@ func (dia *deploymentInitAwaiter) errorMessages() []string {
 	}
 
 	if dia.currentGeneration == "1" {
+		if !dia.pvcsAvailable {
+			messages = append(messages,
+				"Attepted to bind all PersistentVolumeClaims, but not all bindings were attained")
```

Comment:
How about "Failed to bind PersistentVolumeClaim(s)"?

It would also be nice if we could print a list of PVCs that failed to bind.

Reply:
Good idea! I've rephrased the error message, and now include the list of PVCs that failed to bind.